e2e-tests/insurance/cypress/e2e/journeys/export-contract/how-was-the-contract-awarded/how-was-the-contract-awarded-validation.spec.js

```diff
@@ -0,0 +1,109 @@
+import { field as fieldSelector, radios } from '../../../../../../pages/shared';
+import { MAXIMUM_CHARACTERS } from '../../../../../../constants';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import { EXPORT_CONTRACT_FIELDS as FIELDS } from '../../../../../../content-strings/fields/insurance';
+import { EXPORT_CONTRACT as EXPORT_CONTRACT_FIELD_IDS } from '../../../../../../constants/field-ids/insurance/export-contract';
+
+const {
+  ROOT,
+  EXPORT_CONTRACT: { HOW_WAS_THE_CONTRACT_AWARDED },
+} = INSURANCE_ROUTES;
+
+const {
+  HOW_WAS_THE_CONTRACT_AWARDED: { AWARD_METHOD, OTHER_AWARD_METHOD },
+} = EXPORT_CONTRACT_FIELD_IDS;
+
+const AWARD_METHOD_OPTIONS = FIELDS.HOW_WAS_THE_CONTRACT_AWARDED[AWARD_METHOD].OPTIONS;
+
+const {
+  INSURANCE: {
+    EXPORT_CONTRACT: { HOW_WAS_THE_CONTRACT_AWARDED: ERROR_MESSAGES_OBJECT },
+  },
+} = ERROR_MESSAGES;
+
+const OTHER_AWARD_METHOD_MAXIMUM_CHARACTERS = MAXIMUM_CHARACTERS.EXPORT_CONTRACT.OTHER_AWARD_METHOD;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Export contract - How was the contract awarded page - form validation', () => {
+  let referenceNumber;
+  let url;
+
+  before(() => {
+    cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
+      referenceNumber = refNumber;
+
+      // go to the page we want to test.
+      cy.startInsuranceExportContractSection({});
+
+      url = `${baseUrl}${ROOT}/${referenceNumber}${HOW_WAS_THE_CONTRACT_AWARDED}`;
+
+      cy.assertUrl(url);
+    });
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+  });
+
+  after(() => {
+    cy.deleteApplication(referenceNumber);
+  });
+
+  describe('when no radios are selected', () => {
+    beforeEach(() => {
+      cy.navigateToUrl(url);
+    });
+
+    it('should display validation error', () => {
+      const expectedErrorsCount = 1;
+      const expectedErrorMessage = ERROR_MESSAGES_OBJECT[AWARD_METHOD].IS_EMPTY;
+
+      const radioField = {
+        ...fieldSelector(AWARD_METHOD),
+        input: fieldSelector(AWARD_METHOD_OPTIONS.OPEN_TENDER.VALUE).input,
+      };
+
+      cy.submitAndAssertRadioErrors({
+        field: radioField,
+        expectedErrorsCount,
+        expectedErrorMessage,
+      });
+    });
+  });
+
+  describe(`when ${AWARD_METHOD_OPTIONS.OTHER.TEXT} is selected`, () => {
+    const selector = radios(AWARD_METHOD_OPTIONS.OTHER.VALUE).option;
```

Comment:
could these be dereferenced on the top?

Reply:
Will look into this in the next PR since this is approved